src/js/disability-benefits/containers/DocumentRequestPage.jsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { withRouter } from 'react-router';
+import { connect } from 'react-redux';
+import moment from 'moment';
+import AskVAQuestions from '../components/AskVAQuestions';
+import AddFilesForm from '../components/AddFilesForm';
+import Loading from '../components/Loading';
+
+import {
+  addFile,
+  removeFile,
+  submitFiles,
+  getTrackedItem,
+  updateField,
+  showMailOrFaxModal,
+  cancelUpload,
+  getClaimDetail,
+  setFieldsDirty
+} from '../actions';
+
+class DocumentRequestPage extends React.Component {
+  componentWillReceiveProps(props) {
+    if (props.uploadComplete) {
+      this.goToFilesPage();
+    }
+  }
+  goToFilesPage() {
+    this.props.getClaimDetail(this.props.claim.id);
+    this.props.router.push(`your-claims/${this.props.claim.id}/files`);
+  }
+  render() {
+    let content;
+
+    if (this.props.loading) {
+      content = <Loading/>;
+    } else {
+      const trackedItem = this.props.trackedItem;
+      content = (
+        <div className="claim-container">
+          {this.props.uploadError
+            ? <div className="upload-error usa-alert usa-alert-error">
+              <div className="usa-alert-body">
+                <h3 className="usa-alert-heading">Error uploading Files</h3>
+                <p className="usa-alert-text">There was an error uploading your files. Please try again</p>
+              </div>
+            </div>
+            : null}
+          <h1>{trackedItem.displayName}</h1>
+          {trackedItem.overdue && trackedItem.type.endsWith('you_list')
+            ? <div>
+              <h6 className="past-due"><i className="fa fa-exclamation-triangle"></i> Needed from you</h6>
+              <span className="past-due"> - due {moment(trackedItem.suspenseDate).fromNow()}</span>
+            </div>
+            : null}
+          {!trackedItem.overdue && trackedItem.type.endsWith('you_list')
+            ? <div>
+              <h6 className="due-file"><i className="fa fa-exclamation-triangle"></i> Needed from you</h6>
+              <span className="due-file"> - due {moment(trackedItem.suspenseDate).fromNow()}</span>
```

Comment:
This HTML is very similar to the above, could they be collapsed into a single conditional on `trackedItem.type.endsWith('you_list')` and have the classes determined by the `overdue` flag?